lib/DDG/Spice/Symbolab.pm

```diff
@@ -0,0 +1,23 @@
+package DDG::Spice::Symbolab;
+
+use DDG::Spice;
+
+spice is_cached => 1;
+
+name "Symbolab";
+description "Get a solution to a math problem from symbolab.com";
+primary_example_queries "compute integral of x^2", "solve x+2>3x";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Symbolab.pm";
+
+spice to => 'https://www.symbolab.com/ddg?query=$1';
+spice wrap_jsonp_callback => 1;
+
+triggers startend => "calculate", "solve", "compute";
+handle remainder => sub {
+    if ($_ =~ /[\^\*\=\+\-\/\\]/ or $_ =~ /\b(sin|cos|tan|cot|csc|sec|ln|log|sqrt|integral)/i) {
```

Comment:
>Anything you can input in symbolab.com input box will be handled as well by DDG.

Correct, but it seems like the regex you have in place is a little too narrow, so it's reducing what the Instant Answer can do right now? It seems like there's a lot I can do on Symbolab.com that can't be done with the Instant Answer, and I'd like to be able to use as much of your API as we can!

> Anything beyond that will require some additional support. 

Understood, I don't mean for you to change your API at all. I just think the regex we're using here needs to be improved to let more queries through. 

It would be good to have lots of example queries so I can tests this well and especially so we can add them to the Symbolab Instant Answer Page: https://duck.co/ia/view/symbolab so others will be able to see what this Instant Answer can do :)